src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return "Color: " + color + ", Number: " + number;
+    }
```

Comment:
Pay attention to access modifiers. The toString() method should be public to be accessible from outside the class.